media/css/m24/components/navigation-refresh.scss

```diff
@@ -364,6 +364,11 @@ $margin-top: 54px; // top margin offset for mobile navigation menu
         text-decoration: none;
     }
 
+    // band aid fix for https://github.com/mozilla/bedrock/issues/15873
```

Comment:
BTW I just checked if the type stack +vars in MZPv20 and simplified base fixed anything about this quirk but unfortunately it'd still exhibit, so for now this has to stay:/

(There seems to be more unwanted interaction between firefox-base specifics and global components having to resist them, not unlike https://github.com/mozilla/bedrock/issues/15803#issuecomment-2845902837 …)